server/app/policies/library_entry_policy.rb

```diff
@@ -10,10 +10,13 @@ def show?
     true
   end
 
+  def create?
+    !user.nil?
+  end
```

Comment:
Yeah, missed that somehow...

Will think on it overnight, that'll be an interesting one to fix.